src/app/pages/data-protection/components/data-protection-dashboard/data-protection-dashboard.component.ts

```diff
@@ -115,7 +118,7 @@ export class DataProtectionDashboardComponent implements OnInit {
 
     merge(
       this.modalService.onClose$,
-      this.slideInService.onClose$,
+      this.afterClosedSlideIn$.pipe(mergeMap((afterClosed$) => afterClosed$)),
```

Comment:
This is perhaps a bad example for this. I think in the future, this component would be split up into smaller card components and each card will responsible for reloading itself once slideIn is closed as opposed to having this method.